addons/osfstorage/routes.py

```diff
@@ -21,6 +21,15 @@
             json_renderer,
         ),
 
+        Rule(
+            [
+                '/<guid>/osfstorage/quota_status/',
```

Comment:
I don't really see why this to be a v1 endpoint. For `api/wb/views.py:FileMetadataView` you can see we have a good V2 endpoint that handles WB callbacks in a more typical V2 fashion, with standard permission etc.

Reply:
I suggested `v1` mostly for speed; drf adds about 300ms to the fastest of responses, and I think a simple v1 endpoint is much faster. This is going to be called for every file write operation, so a little extra speed will go a long way. Plus the permissions aren't really that urgent; there's not a lot of private information to be gleaned from this endpoint.

Reply:
@felliott Ok, so if speed is really an issue for this (and I assume that's the reason this isn't just yet another check added to `get_auth`,) are we making these callbacks in parallel asynchronously and then awaiting both responses? Because two round trips here seems like it would actually take longer.

Reply:
It's not another call on `get_auth` because `get_auth` is only called once for any set of multiple file operations, and we need this to happen for each file.